codegen/java/src/main/kotlin/io/spine/protodata/codegen/java/file/BeforePrimaryDeclaration.kt

```diff
@@ -32,7 +32,9 @@ import io.spine.protodata.renderer.LineNumber
 import io.spine.protodata.renderer.LineNumber.Companion.notInFile
 import java.lang.System.lineSeparator
 
-private val pattern = Regex("(public\\s+)?(final\\s+)?(abstract\\s+)?(class)|(@?interface)")
+private val pattern = Regex(
```

Comment:
Please comment on the purpose — so that any potential changes and-or mistakes could be corrected in accordance with the initial thought that you have.